third_party/amd/backend/compiler.py

```diff
@@ -243,7 +243,7 @@ def make_ttgir(mod, metadata, options):
         passes.ttgpuir.add_reduce_data_duplication(pm)
         if amd.has_matrix_core_feature(options.arch):
             amd.passes.ttgpuir.add_reorder_instructions(pm)
-            use_block_pingpong = os.getenv("TRITON_HIP_USE_BLOCK_PINGPONG", "0") == "1"
+            use_block_pingpong = os.getenv("TRITON_HIP_USE_BLOCK_PINGPONG", "1") == "1"
```

Comment:
It's fine to enable it on by default to test out this pull request. But we should turn it off before landing and have a separate pr to turn it on explicitly.

Reply:
done